README.md

```diff
@@ -39,31 +39,89 @@ This project is organized in a monorepo structure and contains the following wor
 ## What should I do when I ...?
 
 #### want to work with the panel
+
 - Run `docker-compose up -d` to start the database.
 - Run `yarn w api dev` to start the server in development mode.
 - Run `yarn w api panel` to start the panel in development mode.
 
 #### want to work with the api
+
 - Run `docker-compose up -d` to start the database.
 - Run `yarn w api dev` to start server in development mode.
 
 #### want to work with the website
+
 - Run `yarn w website dev` to start the website in development mode.
 
 #### want to create new components
+
 - Run `yarn w ui dev` to run the storybook and see all component stories.
 
 #### have troubles when running some script
+
 - Run `yarn clean` to clean move the entire project to a clean state.
 - Run `yarn` to install dependencies.
 - Run `yarn bootstrap` to populate the project again.
 - Check StackOverflow and GitHub pasting the error message you got.
 - Write on a development channel on our Slack.
 
 ## Unified scripts
+
 To increase project consistency, some standard scripts are unified across all workspaces.
 
 - **dev** - runs the package in a local development mode.
 - **build** - creates production build of the package.
 - **start** - runs previously built package locally.
 - **test** - runs tests for the package.
+
+## PR naming rules
+
+We have very precise rules over how our PR title can be formatted. This leads to more readable messages that are easy to follow when looking through the project history.
+
+### PR Title Format
+
+The PR title has a special format that includes a **type**, a **scope** and a **subject**:
+
+```
+<type>(<scope>): <subject>
+```
+
+The **type** and **subject** is mandatory but the **scope** is optional.
+
+Samples:
+
+```
+feat(ui): Add Icon component.
+```
+
+```
+fix: Correct minor typos in code
+```
+
+```
+docs: Fix reference to Conventional Commits
+```
+
+### Type
+
+Must be one of the following:
+
+- **build**: Changes that affect the build system or external dependencies
+- **ci**: Changes to our CI configuration files and scripts
+- **docs**: Documentation only changes
+- **feat**: A new feature
+- **fix**: A bug fix
+- **perf**: A code change that improves performance
+- **refactor**: A code change that neither fixes a bug nor adds a feature
+- **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+- **test**: Adding missing tests or correcting existing tests
+
+### Scopes
+
+- **core** - Changes in the root directory
```

Comment:
For consistency with the previous point, I would use `:` instead of `-` (or use `-` in the previous point).

Also it would be great to add descriptions to all scopes. My suggestion is:
**core** - Changes made only in the root directory.
**api** - Changes made only in the api and/or shared directory.
**panel** - Changes made only in the panel and/or shared directory.
**website** - Changes made only in the website and/or shared directory.
**ui** - Changes made only in the ui and/or shared directory.